cypress/support/e2e.js

```diff
@@ -0,0 +1,20 @@
+// ***********************************************************
```

Comment:
And this?

Reply:
I don't believe this file is being used or is necessary.  Since it's in the root of the entire project and not the test project

Reply:
Great!